### DISCUSSION TITLE
QtWebEngine crash

### DISCUSSION THREAD

**Discussion by mph070770** [cgiesche/streamdeck-homeassistant#201] [Help and Support]
It seems that it's an issue reported by a number of users:

https://www.reddit.com/r/elgato/comments/zai0rb/qtwebengineprocess_crashes_since_upgrading_stream/

I'm using 6.4.1 of streamdeck on Mac Sonoma and the latest 2.4.1 of your plug in.

I was happily using your plugin with Ventura (not sure if it was an earlier version of StreamDeck SW).

Any idea of a fix?

**Comment by cgiesche** [comment]
Hello and thanks for using my plugin!

My plugin (better said, javascript plugins in general) are executed within a QtWebEngine by the StreamDeck application. From my perspective, i can not do anything about the problem. When the container (WebEngine) crashes, my plugin instance is gone. The plugin can not access anything outside the container or the container itself.

If there are other users/plugins affected, it seems we have to wait for elgato to fix the problem. :(
